core/src/global.rs

```diff
@@ -155,10 +156,14 @@ where
 	let chain_type = CHAIN_TYPE.read().clone();
 	match chain_type {
 		// Mainnet has Cuckaroo29 for AR and Cuckatoo30+ for AF
+		ChainTypes::Mainnet if edge_bits == 29 && valid_header_version(_height, HeaderVersion::new(2))
+						=> new_cuckarood_ctx(edge_bits, proof_size),
```

Comment:
I think you can rename `_height` to `height` since we use it now. Also Cuckatoo31+ instead of Cuckatoo30+?

Reply:
thanks for careful reading. fixed now.